### PR TITLE
 Add The Faith Militant preview cards

### PR DESCRIPTION
**Pull request by DukeTax**
https://www.fantasyflightgames.com/en/news/2018/1/3/the-faith-militant/

https://scontent-amt2-1.xx.fbcdn.net/v/t1.0-9/26167994_1828364043905651_771413969039201790_n.png?oh=fd0c2690205bc90658af5cf9df2d652d&oe=5AFAF4E5

### CODE REVIEW

File: pack/TFM.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        "is_power": true,
+        "is_unique": true,
+        "name": "ColdHands",
+        "octgn_id": null,
+        "pack_code": "TFM",
+        "position": 85,
+        "quantity": 3,
+        "strength": 3,
+        "text": "Coldhands cannot be saved.\n<b>Reaction:</b> After you marshal Coldhands, choose another non-<i>Army</i> character controlled by each player. Remove each of those characters from the game until Coldhands leaves play.",
+        "traits": "Spy.",
```
Comment (brine) [inline pack/TFM.json]: traits is "The North."

File: pack/TFM.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        "deck_limit": 3,
+        "designer": "Card design by 2011 Joust World Champion, Brett Zeiler.",
+        "faction_code": "thenightswatch",
+        "illustrator": "",
+        "is_intrigue": false,
+        "is_loyal": false,
+        "is_military": false,
+        "is_power": true,
+        "is_unique": true,
+        "name": "ColdHands",
```
Comment (brine) [inline pack/TFM.json]: the H shouldn't be capitalized

File: pack/TFM.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
+[
+    {
+        "code": "08085",
+        "cost": 5,
+        "deck_limit": 3,
+        "designer": "Card design by 2011 Joust World Champion, Brett Zeiler.",
+        "faction_code": "thenightswatch",
+        "illustrator": "",
```
Comment (brine) [inline pack/TFM.json]: illustrator is "Diego Gisbert Llorens", based off the original CCG version of the card with the same artwork

**Comment by DukeTax** [comment]
Thanks for the review.

**Comment by ystros** [comment]
@Alsciende thanks for making me a collaborator a while back. Let me know if you want me to stick to code reviews or if I should merge things as well after they've been reviewed.

**Comment by Alsciende** [comment]
@ystros I think it's better if I do the merge, because the deployment is not automatic. If the PR is merged and closed, I'll forget I have to deploy the data.

**Comment by Alsciende** [comment]
Thanks for your contribution @DukeTax and @ystros  :)

**Pull request merged**